client/tests/conftest.py

```diff
@@ -149,24 +149,24 @@ def s3(aws_credentials):
 
 @pytest.fixture(scope="module")
 def local_provider_source():
-        def get_local():
-            ff.register_user("test_user").make_default_owner()
-            provider = ff.register_local()
-            source = provider.register_file(
-                name="transactions",
-                variant="quickstart",
-                description="A dataset of fraudulent transactions.",
-                path=f"{dir_path}/test_files/input_files/transactions.csv"
-            )
-            return (provider, source)
-        
-        return get_local
+    def get_local(_):
```

Comment:
Is this underscore supposed to be here?

Reply:
Yes, previously, it was a named param that I don't actually need for the local fixture, but to ensure I had the same function signature for both local and hosted, I passed the param here, but changed it to an underscore to convey the fact I didn't intend to use it.